gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -65,6 +73,113 @@ public RoomLocalDataStore(GndApplication app) {
             .build();
   }
 
+  @Override
+  public Completable insertOrUpdateOption(String fieldId, Option option) {
```

Comment:
Same comment as before; only the top level insertOrUpdateProject needs to be public.